samples/nextjs/package.json

```diff
@@ -10,6 +10,7 @@
     ],
     "sitecoreConfigPath": "/App_Config/Include/zzz",
     "graphQLEndpointPath": "/sitecore/api/graph/edge",
+    "trackingEndpointPath": "/api/track",
```

Comment:
Is it `/api/track` or `/sitecore/api/track`? I see the latter in `JssNextWeb.config` comments and the default in `tracking-service.ts`.

Reply:
The default is `/sitecore/api/track`.
But because of the client IP issue, we cannot use URL rewrites in next.js and have to rely on `/pages/api` endpoints.
That is why it is changed to start from `/api` (I hope temporary).
Other frameworks may not suffer from this issue and can use `/sitecore/api/track`.